### ISSUE TITLE
Error 1: [Code 0] Internal error. Some validation steps may not have occurred

### ISSUE THREAD

**Issue by franklin-feingold** [bids-standard/legacy-validator#1010]
A user originally identified a validation bug when trying to upload onto OpenNeuro. The user is trying to upload from a remote server, using a Chrome browser. This error has since been replicated when using the web version of the validator

```
Error: 1
Internal error. SOME VALIDATION STEPS MAY NOT HAVE OCCURRED
 
REASON:
Cannot read property 'trim' of undefined; please help the BIDS team and community by opening an issue at (https://github.com/bids-standard/bids-validator/issues) with the evidence here.
 
EVIDENCE:
TypeError: Cannot read property 'trim' of undefined at Me (https://openneuro.org/0-dd64aa8d.bundle.worker.js:69:100962) at https://openneuro.org/0-dd64aa8d.bundle.worker
```

Dataset details:
2506 files, 165.67 GB, 56 subjects, 1 session, task based fMRI with 2 tasks, modalities: sbref, bold, events, T1w, epi

**Comment by rwblair** [comment]
@franklin-feingold can you check with the users that all their events tsv files have both onset and duration headers? First hunch is that this is where the error may be coming from.

**Comment by GabrielPelletier** [comment]
@rwblair @franklin-feingold Indeed the problem was with one of the events.tsv files. For some reason a single one of them in my entire data set was empty. I corrected this and the validation now runs smoothly with no errors returned. Thanks

**Comment by rwblair** [comment]
fixed by #1012
